auxiliary_files/config_files/mumble-server.service.in

```diff
@@ -1,10 +1,10 @@
 [Unit]
-Description=Mumble Daemon
+Description=Mumble server
 After=network.target
 
 [Service]
 Type=exec
-ExecStart=/usr/bin/murmurd -fg -ini /etc/murmur.ini
+ExecStart=/usr/bin/@MUMBLE_SERVER_BINARY_NAME@ -fg -ini /etc/mumble-server.ini
```

Comment:
Here we are running as root, but ideally we want to not do that but instead run as a separate system user (e.g. `mumble-server`, which is also referenced in the logrotate config).

For this
```
User=mumble-server
Group=mumble-server
```

should be set in this service file.

To create the user and group on the target system with the help of [sysusers.d](https://man.archlinux.org/man/core/systemd/sysusers.d.5.en), have a look at what we're doing [here](https://github.com/archlinux/svntogit-community/blob/ba167269f8336659313d589dea65642b4eae4f85/trunk/murmur.sysusers#L1) for the creation of the `murmur` user (you should omit the UID, as it is specific to Arch and sysusers.d will pick a random one, which is better).

Reply:
FWIW, I am shipping a modified systemd service file that has more hardening options enabled: https://github.com/archlinux/svntogit-community/blob/ba167269f8336659313d589dea65642b4eae4f85/trunk/murmur.service

Reply:
I took the liberty of essentially copying your service file (making only minor adaptions)